.github/workflows/build.yml

```
@@ -1,19 +1,25 @@
 name: build
 
-on: [push, create]
+on: [push, create, pull_request]
 
 jobs:
   build_matrix:
     strategy:
       matrix:
-        os: ['ubuntu-18.04', 'ubuntu-20.04']
-        ruby: ['2.6', '2.7', '3.0', '3.1']
+        os: ['ubuntu-18.04', 'ubuntu-20.04', 'ubuntu-latest']
```

```suggestion
        os:
          - "ubuntu-18.04"
          - "ubuntu-20.04"
          - "ubuntu-latest"
```

What do you think about keeping consistency with the way how Ruby versions are defined below?